.github/pull_request_template.md

```diff
@@ -3,6 +3,10 @@
 Provide a high-level description of this change.   
 -->
 
+## Functional Demo
```

Comment:
Are we sure this doesn't significantly increase the effort of contributors to add new plugins?